state_machine/applications/flight/lib/logs.py

```diff
@@ -40,10 +43,11 @@ def beacon_packet():
 
 
 def human_time_stamp():
-    """Returns a human readable time stamp in the format: 'year.month.day hour:min'
+    """Returns a human readable time stamp in the format: 'boot_year.month.day_hour:min'
```

Comment:
boot.year.month.day_hour:min:sec, seems to be what you are doing

Reply:
The RTC driver won't accept -1 for the weekday (it becomes 255 because it's treating it as unsigned).